src/main/java/frc/robot/Constants.java

```diff
@@ -12,4 +11,26 @@
  * <p>It is advised to statically import this class (or one of its inner classes) wherever the
  * constants are needed, to reduce verbosity.
  */
-public final class Constants {}
+public final class Constants {
+    // motor controller 
+    public static final int k_back_left_motor = 1;
```

Comment:
General incorrect format.
Constants should always be prefixed with `k` but also named in camel case rather than snake.

```suggestion
    public static final int kBackLeftMotor = 1;
```